Source/KeyHandler.mm

```diff
@@ -1276,11 +1358,11 @@ - (void)_walk
 
     // if DEBUG is defined, a GraphViz file is written to kGraphVizOutputfile
 #if DEBUG
-    std::string dotDump = _builder->grid().dumpDOT();
-    NSString *dotStr = [NSString stringWithUTF8String:dotDump.c_str()];
-    NSError *error = nil;
-
-    BOOL __unused success = [dotStr writeToFile:kGraphVizOutputfile atomically:YES encoding:NSUTF8StringEncoding error:&error];
+//    std::string dotDump = _builder->grid().dumpDOT();
```

Comment:
Should this be uncommented, or should the entire `#if DEBUG` section be deleted?